locales/en-US/strings.properties

```diff
@@ -1,29 +1,39 @@
-top_sites=Top Sites
-highlights=Highlights
-visited=Visited
-bookmarked=Bookmarked
-synced=Synced from another device
-open=Open
-topic=Topic
-bookmark=Bookmark
-remove_bookmark=Remove Bookmark
-share=Share
-copy_address=Copy Address
-email_link=Email Link...
-open_new_window=Open in a New Window
-open_private_window=Open in a Private Window
-dismiss=Dismiss
-delete=Delete from History
+header_top_sites=Top Sites
+header_highlights=Highlights
+type_label_visited=Visited
+type_label_bookmarked=Bookmarked
+type_label_synced=Synced from another device
+type_label_open=Open
```

Comment:
"Open" does seem a bit ambiguous here especially that the previous ones weren't "Visit" and "Bookmark" so this could have been "Opened". Perhaps we could just include it as an example in the above "LOCALIZATION NOTE (type_label_*): ...<general description>... E.g., the type indicates that the page is bookmarked or is currently open."